quicktube.js

```diff
@@ -1,283 +1,518 @@
-            }
-        },
-
-        getIframePlayer: function(id, parent, parentId) {
-            var self = this;
-            var src = self._domain + src + self._settings;
-            var iframe = document.createElement("iframe");
-            iframe.src = self._domain + id + self._settings;
-            iframe.width = "100%";
-            iframe.id = parentId;
-            iframe.className = this.className;
-            return $(iframe);
-        },
-
-        stopVideo: function(parentId) {
-            var self = this;
-            var $parent = $("[data-quicktube=\"" + parentId + "\"]");
-            var frame = $parent.find("iframe");
-            var func = "pauseVideo";
-
-            if(!this.jamesPlayer) {
-                return;
-            }
+    getIframePlayer(id, parent, parentId) {
+        let iframe = document.createElement('iframe');
+        iframe.src = this._domain + id + this._settings;
+        iframe.width = '100%';
+        iframe.id = parentId;
+        iframe.className = this.iframeClass;
+        return iframe;
+    },
 
-            this.jamesPlayer.pauseVideo();
-            $parent.removeClass(self.activeClass).addClass(self.pausedClass);
-            self.showPosterFrame($parent.find("[data-quicktube-poster]"));
-            $parent.data("video-playing", false);
-            self._players[parentId] = false;
-            $(window).trigger("quicktube:pause", parentId, $parent);
-        },
-
-        trackEvent: function (event) {
-            if (typeof window._gaq === "object") {
-                window._gaq.push(["_trackEvent", event.eventCategory, event.eventAction, event.eventLabel]);
-            } else if (typeof window.ga === "function") {
-                window.ga('send', 'event', event.eventCategory, event.eventAction, event.eventLabel);
-            } else {
-            }
-        },
-    };
-
-    // Export this to window directly.
-    window.onYouTubeIframeAPIReady = function() {
-        $(document).ready(function() {
-            QT.init();
-        });
-    };
-
-    return QT;
-}));
+    stopVideo(parentId) {
+        let playerEl = document.querySelector(`[data-quicktube='${parentId}']`);
+
+        if(!this.quicktubePlayer) {
+            return;
+        }
+
+        this.quicktubePlayer.pauseVideo();
+        playerEl.classList.remove(this.activeClass);
+        playerEl.classList.add(this.pausedClass);
+        this.showPosterFrame(playerEl.querySelector('[data-quicktube-poster]'));
+        playerEl.getAttribute('data-video-playing');
+        this._players[parentId] = false;
+        window.dispatchEvent(new Event('quicktube:pause'));
+    },
+
+    trackEvent(event) {
+        if (typeof window._gaq === 'object') {
+            window._gaq.push(['_trackEvent', event.eventCategory, event.eventAction, event.eventLabel]);
+        } else if (typeof window.ga === "function") {
+            window.ga('send', 'event', event.eventCategory, event.eventAction, event.eventLabel);
```

Comment:
I trust you on this without explicitly knowing why. Have simply left the `window.ga === 'function'` condition